eZ/Publish/Core/Persistence/Tests/TransformationProcessor/_fixtures/transformations/latin.tr

```diff
@@ -194,12 +200,12 @@ U+017E = "z"
 # Transliteration of latin1, is used in URLs and identifiers
 latin1_transliterate_ascii:
 # Turn some latin1 characters into ASCII equivalents
-U+00E6 = "ae" # � => ae
-U+00C6 = "AE" # � => AE
-U+00E5 = "aa" # � => aa
-U+00C5 = "AA" # � => AA
-U+00F8 = "oe" # � => oe
-U+00D8 = "OE" # � => OE
+U+00E6 = "ae" # � => ae
```

Comment:
How is it related to this PR?

Reply:
It's possible that you reverted intended changes or your client changed line endings. Simplest is to get rid of these unrelated changes, by taking it from master on these chunks.